aten/src/ATen/Config.h.in

```diff
@@ -10,3 +10,9 @@
 #define AT_MKL_ENABLED() @AT_MKL_ENABLED@
 #define AT_NNPACK_ENABLED() @AT_NNPACK_ENABLED@
 #define CAFFE2_STATIC_LINK_CUDA() @CAFFE2_STATIC_LINK_CUDA_INT@
+#define AT_BUILD_WITH_BLAS() @USE_BLAS@
+
+#ifdef FBCODE_CAFFE2
+#undef AT_BUILD_WITH_BLAS
+#define AT_BUILD_WITH_BLAS() 1
+#endif
```

Comment:
@anjali411 I have fixed the OSS bazel build. However, does the internal build also use bazel, if so, I think you need to change the internal BUILD.bazel as I did in https://github.com/pytorch/pytorch/pull/30898/files#diff-0ea22445d900db774c8deb00f4a4df55R519, instead of having these lines here.

Reply:
yeah modified the internal BUILD.bazel